status/worksets.py

```diff
@@ -38,7 +38,7 @@ def get(self, worksets='all'):
         t = self.application.loader.load("worksets.html")
         ws_data=self.worksets_data()
         headers= [['Date Run', 'date_run'],['Workset Name', 'workset_name'], \
-                 ['Projects (samples)','projects'], ['Operator', 'technician'],\
+                 ['Projects (samples)','projects'], ['Date finished', 'finish date'],['Operator', 'technician'],\
```

Comment:
no it can't, the view is 'finish date' without a _

On Fri, Jul 10, 2015 at 9:17 AM, senthil10 <notifications@github.com> wrote:

> In status/worksets.py
> <https://github.com/SciLifeLab/genomics-status/pull/215#discussion_r34332323>
> :
>
> > @@ -38,7 +38,7 @@ def get(self, worksets='all'):
> >          t = self.application.loader.load("worksets.html")
> >          ws_data=self.worksets_data()
> >          headers= [['Date Run', 'date_run'],['Workset Name', 'workset_name'], \
> > -                 ['Projects (samples)','projects'], ['Operator', 'technician'],\
> > +                 ['Projects (samples)','projects'], ['Date finished', 'finish date'],['Operator', 'technician'],\
>
> should it be finish_date with a _ ? (not a stopper, just asking as every
> other dual word key have a _) [image: :smile_cat:]
>
> Maybe I can merge and you can change it later if needed ? [image: :eyes:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/genomics-status/pull/215/files#r34332323>.
>